examples/module1/lesson5/tanstack-query/components/ArticleAuthors.tsx

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from '@tanstack/react-query';
+
+function ArticleAuthors() {
+  const { isPending, error, data } = useQuery({
```

Comment:
rozumiem że `isPending` po to żeby wyświetlać ładowanie w momencie kiedy z innego miejsca apki zinvalidujesz te zapytanie, myślę tylko czy to nie sprawia że wtedy za każdym razem pokaże się `loading...` i pokaże zaktualizowane artykuł dopiero po loaderze, korzystając z `isLoading` pokazuje on się tylko przy inicjalnym załadowaniu query a później już nie więc do aktualnie wyświetlanych danych powinien tylko `dospawać` się nowy artykuł

myślę że z perspektywy UI może to być lepsze, ale spróbuj bo nie mam jak odpalić tego i może coś źle pisze ;p